app/model/DisplayRepository.scala

```diff
@@ -21,19 +21,20 @@ case class DisplayRepository(
     * Indicates whether a dependency version is up-to-date with other repositories.
     *
     * @param dependency the dependency id
+    * @param subfolder  the dependency subfolder
     * @return True when the dependency is up to date
     */
-  def isVersionUpToDate(dependency: String): Boolean = {
-    val version = getDependencyVersion(dependency)
+  def isVersionUpToDate(dependency: String, subfolder: String): Boolean = {
```

Comment:
it's debatable, but I think this method and the others would be be more maintainable if we were passing a `Dependency` object instead of `dependency` and `subfolder`

Reply:
yeah it's true. It's not so essential as some methods like getLocalDependencyVersion or getCentralDependencyVersion need the dependency name and others the complete dependency.
I made the change.